Examples/destination_plugins/AppsFlyerDestination.swift

```diff
@@ -0,0 +1,352 @@
+//
+//  AppsFlyerDestination.swift
+//
+//  Created by Alan Charles on 6/22/21.
+//
+// NOTE: You can see this plugin in use in the DestinationsExample application.
+//
+// This plugin is NOT SUPPORTED by Segment.  It is here merely as an example,
+// and for your convenience should you find it useful.
+//
+// AppsFlyer SPM package can be found here: https://github.com/adjust/ios_sdk
+// MIT License
+//
+// Copyright (c) 2021 Segment
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+// *** To Implement Deep Linking functionality reference: https://support.appsflyer.com/hc/en-us/articles/208874366 ****
+
+import Foundation
+import Segment
+import AppsFlyerLib
+import  UIKit
+
+internal struct AppsFlyerSettings: Codable {
+    let appsFlyerDevKey: String
+    let appleAppID: String
+    let trackAttributionData: Bool?
+}
+
+@objc
+class AppsFlyerDestination: UIResponder, DestinationPlugin, RemoteNotifications, iOSLifecycle  {
+    
+    let timeline: Timeline = Timeline()
+    let type: PluginType = .destination
+    let name: String
+    var analytics: Analytics?
+    
+    internal var settings: AppsFlyerSettings? = nil
+    
+    required init(name: String) {
+        self.name = name
+        analytics?.track(name: "AppsFlyer Loaded")
+    }
+    
+    public func update(settings: Settings) {
+        
+        guard let settings: AppsFlyerSettings = settings.integrationSettings(name: "AppsFlyer") else {return}
```

Comment:
`{ return }` - needs spaces, not usually a stickler for this kind of stuff but it looks so much cleaner than none.

Reply:
lol yeah no problem at all